DESIGN_DOCS.md

```diff
@@ -1,13 +1,26 @@
 # Design Documents
 
+* [Design Doc Template](https://github.com/LibertyDSNP/meta/blob/main/templates/DESIGN_DOC_TEMPLATE.md)
+
 ## Purposes of this document
 * To describe what our design documents are for, their structure, and what goes in them.
 * To describe how DSNP Design Documents are produced and where they're published.
 
-## What Is a Design Document?
-It's a document clearly describing a problem, a solution to that problem, and why we think the proposal is the right solution.
+## What is a Design Document?
+It's a document clearly describing a problem or feature, a solution for it, and why we think the proposal is the right solution.
+We'll use _"problem"_ here to broadly mean "anything that means we need to write some code."
 It has enough information to begin writing stories for implementing the solution in the document.
- 
+
+## When do we expect a Design Document?
+Generally we expect to see a Design Document when the solution is neither simple nor obvious to a developer with domain knowledge of the problem.
+If the solution will result in more than one reasonably-sized Issue, that is a sign a Design Document might be needed.
+
+**Examples that don't need a Design Document:** Major revisions to a README file, bug fixes (usually), implementing a login screen.
```

Comment:
These were some changes sitting in my local copy I'd forgotten about, but had been asked to add.